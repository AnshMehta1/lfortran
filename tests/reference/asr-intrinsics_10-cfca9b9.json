{
    "basename": "asr-intrinsics_10-cfca9b9",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/intrinsics_10.f90",
    "infile_hash": "574d28f0c6b85cf1bd1263ef8530e069cbe07753791f9e8bec6d939e",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-intrinsics_10-cfca9b9.stdout",
    "stdout_hash": "d9e9a91a355db381854f933762fa976ac64985c4e96468bcf0b8ffef",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}