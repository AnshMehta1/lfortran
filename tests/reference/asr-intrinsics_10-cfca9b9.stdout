(TranslationUnit
    (SymbolTable
        1
        {
            intrinsics_10:
                (Program
                    (SymbolTable
                        2
                        {
                            x:
                                (Variable
                                    2
                                    x
                                    []
                                    Local
                                    (IntegerConstant 3 (Integer 4))
                                    (IntegerConstant 3 (Integer 4))
                                    Parameter
                                    (Integer 4)
                                    ()
                                    Source
                                    Public
                                    Required
                                    .false.
                                )
                        })
                    intrinsics_10
                    []
                    [(Print
                        [(IntrinsicElementalFunction
                            Real
                            [(Var 2 x)]
                            0
                            (Real 4)
                            (RealConstant
                                3.000000
                                (Real 4)
                            )
                        )]
                        ()
                        ()
                    )]
                )
        })
    []
)
